CONTRIBUTOR-LADDER.md

```diff
@@ -155,6 +155,8 @@ rights and responsibilities of a Reviewer.
    * Same as Reviewers
    * Engage/drive in feature proposals and ensure release readiness of the
    repository.
+   * Update/modify project goals and governance policies.
```

Comment:
```suggestion
   * Update/modify project goals and governance policies.
```
Approver in minor repo doing project goals and governance policies? Since were not single repo (or even have a single main repo), this is problematic IMO.

Reply:
hummm, true but our problems is we dont distinguish between minor repo Approvers since we add all of them in maintainers list.

Reply:
Do we distinguish them here though.

Can we move this sentence as is to maintainers? Project goal and policy is not for every approver anyways. Then we can add a repo specific version here, approver can update repository/componentt goals.. I don't think approver needs to worry about governance policies for the repository either?

We may need to consider what and who we call maintainers vs approvers vs owners? Cross-check versus ecosystem etc... I think part of the problem is Github term is maintainer (anyone with write access) and Prow/k8s calls them approvers.

Something to think about anyways, we now have good faith community so it isn't critical.

Reply:
My plan was to add there. But we dont have a separate responsibilities section there as because we add all approvers in maintainers list and we also define maintainer as. `A Metal3 maintainer is a participant in the project with the ability to approve changes proposed to a given repository.` What you are insisting now is to have a new role maintainer and that would sit somewhere on top of Approvers.

Reply:
I agree with your concern though and we can ofcourse discuss and add the separate role on top and distinguish between maintainers and approvers, with regards to current process I believe this should be ok.

Reply:
I'm not insisting, just asking questions :)  

I did take a second look though, and noticed I did make mistake about the maintainer role, as it is not in the same document, but in the next one as separate concept. In review mode it looked like it was already sitting on top of approver. 

We can go with this now, but let's discuss it in future. Multi-repo vs single-repo org does lead to some weird concepts at times. Like said, we're good faith community now so its not an issue.